lsp-mode.el

```diff
@@ -7248,18 +7248,21 @@ CALLBACK is the status callback passed by Flycheck."
                 :line (1+ (lsp-diagnostic-line diag))
                 :column (1+ (lsp-diagnostic-column diag))
                 :message (lsp-diagnostic-message diag)
-                :level (lsp--flycheck-calculate-level diag)
+                :level (if (string-match "clippy" (lsp-diagnostic-message diag))
+                           'info
+                        (lsp--flycheck-calculate-level diag))
```

Comment:
The problem with clippy is that it returns error code 1. I don't like this solution so maybe somebody has a but idea ?

Reply:
I don't understand why returning error code 1 is a problem.